Sources/GRPC/ClientConnection.swift

```diff
@@ -469,6 +469,24 @@ extension ClientConnection {
     @preconcurrency
     public var debugChannelInitializer: (@Sendable (Channel) -> EventLoopFuture<Void>)?
 
+    #if canImport(Network)
+    @available(macOS 10.14, iOS 12.0, watchOS 6.0, tvOS 12.0, *)
+    public var clientBootstrapNWParametersConfigurator: (
```

Comment:
We can drop the `clientBootstrap` prefix here, we hide the concept of a bootstrap. Can you add some docs as well please?